### DISCUSSION TITLE
Compiling the rust9x-sample project

### DISCUSSION THREAD

**Discussion by git-g00d** [rust9x/rust#42] [Q&A]
Hello 👋 , I am having issues getting even the rust9x-sample project to build. The issue I get is this:
```
PS C:\volume> just build --release
cargo +rust9x build --target i586-rust9x-windows-msvc --release
error: 'cargo.exe' is not installed for the custom toolchain 'rust9x'.
note: this is a custom toolchain, which cannot use `rustup component add`
help: if you built this toolchain from source, and used `rustup toolchain link`, then you may be able to build the component with `x.py` 
error: Recipe `r9x` failed on line 21 with exit code 1
```

Trying to copy the default cargo binary to the bin folder for the rust9x toolcahin fails immediately with a recursion error:
```
PS C:\volume> cp C:\Users\ContainerAdministrator\.cargo\bin\cargo.exe C:\rust\rust9x\bin\
PS C:\volume> just build --release
cargo +rust9x build --target i586-rust9x-windows-msvc --release
error: infinite recursion detected 
error: Recipe `r9x` failed on line 21 with exit code 1
```

I've used the pre-built Rust9x v1.76.0-b a couple months ago and do not recall needing to do anything special to use it out of the box. I feel like I am missing something here. Any help would be appreciated!

## Notes
May be worth mentioning I am using Rust9x inside a Docker container; however, I get the same issues when trying on my local system.

**Comment by seritools** [comment]
haven't run into this myself, but i'm working on a 1.83 update for rust9x, so i'll check there again once i get to it
